components/CategoryPage.js

```diff
@@ -2,6 +2,8 @@
 import React, { Component, PropTypes } from 'react';
 import { Link } from 'react-router';
 import Gmap from './resources/Map.js';
+import LargeHeader from './LargeHeader.js'
+import Footer from './Footer.js'
 
 var categories = [];
       
```

Comment:
I'd prefer for all of the components to be created using the same standards. CategoryBox, CategoryList, and Category should probably be using ES6 classes like ContentPage is. It might be a bit nitpicky, and probably doesn't need to block a merge, but I feel like we should do it at some point.